packages/styles/settings-tools/_s.spaces.scss

```diff
@@ -21,12 +21,13 @@ $sizes: (
   1000: $mu1000,
```

Comment:
Maybe we should add support for screens sizes using `modify-from-screens`,
this could look like : 

```scss
@mixin make-space-util($prop, $side, $size, $size-list: ('s')) {
   ...
   .mu-#{$prop}-#{$size} {
      @modify-from-screens($size-list) {
        ...
      }
   }
  ...
}
```

We could also add the `v` and `h` variants in sides.

Reply:
Do i have to add these variation now or is it possible to plan this in a second time ?
We will need the enhancement very very soon. If you agree i prefere to continue without this additionnal enhancement

Reply:
later is ok

Reply:
OK thanks so i remove the WIP status ;)